app/assets/stylesheets/views/_homepage.scss

```diff
@@ -242,3 +250,27 @@
     }
   }
 }
+
+// Created a custom section and header pattern to
+// - display a border-top of 2px (to replace .home-divider)
+// - margin-bottom of 30px (both mobile and desktop)
+// - padding-top to create a gap between border and heading
+
+.homepage-section {
+  margin: govuk-spacing(4) 0 govuk-spacing(9);
+}
+
+.homepage-section__heading {
+  border-top-style: solid;
+  border-top-width: 2px;
+  margin: 0 0 govuk-spacing(6);
+  padding: govuk-spacing(4) 0 0;
+}
+
+.homepage-section__depts {
+  .gem-c-big-number {
+    &:nth-child(1) {
+      margin-bottom: govuk-spacing(6);
+    }
+  }
```

Comment:
Overriding a component's CSS is brittle because it heavily depends on the component staying the same, which isn't something that's guaranteed. Is there another way we can achieve this spacing?

Reply:
I could add a wrapper div / class around the component and target that instead?

Reply:
I think that's a better option - and then could you add a issue on the components gem about adding the ability to set the margin on the component? 

Once that feature has been added we can then remove the wrapping element and add the spacing using the component - but that can be done at a later date.

Reply:
Done. Created an issue here: https://github.com/alphagov/govuk_publishing_components/issues/2479